arviz/tests/test_data.py

```diff
@@ -209,6 +210,30 @@ def get_inference_data(self):
             dims={'theta': ['school'], 'theta_tilde': ['school']},
         )
 
+class TestEmceeNetCDFUtils(BaseArvizTest):
```

Comment:
Not picky and not not a blocker but do want to ask. Do we want tests to be functional style or xunit style?

Reply:
Oof, I figured `BaseArvizTest` contained some extra tests, and I was pleased they all passed. 

What if we merge this, and I'll do a followup converting all the `NetCDFUtils` tests?

Reply:
Feel free to merge! It's great that there are tests. I just moreso wanted to understand where we *eventually* wanted the test style to be.